crates/tx5-go-pion-sys/Cargo.toml

```diff
@@ -47,3 +47,6 @@ force_dynamic_link = []
 # forces the code to run through the static linking process
 # even if that may not be supported by the target platform
 force_static_link = []
+
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(link_dynamic)', 'cfg(link_static)'] }
```

Comment:
thanks!

Reply:
No worries, already ran into this same funky clippy behavior on Holochain so it's an easy fix

From what I've read, they're going to give us cleaner ways of expressing custom cfgs at some point